src/QMCApp/tests/test_QMCDriverFactory.cpp

```diff
@@ -52,7 +103,29 @@ TEST_CASE("QMCDriverFactory::VMCBatchedDriver", "[qmcapp]")
   REQUIRE(okay);
   xmlNodePtr node                           = doc.getRoot();
   QMCDriverFactory::DriverAssemblyState das = driver_factory.readSection(0, node);
-  REQUIRE(das.new_run_type == QMCDriverFactory::QMCRunType::VMC_BATCH);
-  REQUIRE(das.append_run == false);
+  REQUIRE(das.new_run_type == QMCRunType::VMC_BATCH);
+
+  // MinimalParticlePool mpp;
```

Comment:
Is this commented-out code going to be used in the future, or can it be removed?

Reply:
It will be used in the next PR.  This PR is a retcon to ease review.